spring-integration-http/src/main/java/org/springframework/integration/http/config/IntegrationGraphControllerRegistrar.java

```diff
@@ -98,4 +113,22 @@ public void postProcessBeanFactory(ConfigurableListableBeanFactory beanFactory)
 
 	}
 
+	static class IntegrationGraphCorsConfigurer extends WebMvcConfigurerAdapter {
+
+		private final String path;
+
+		private final String[] allowedOrigins;
+
+		public IntegrationGraphCorsConfigurer(String path, String[] allowedOrigins) {
```

Comment:
Redundant modifier. See Travis report
And seems for me the whole class can be `private`.